integrations/symfony-bundle.rst

```diff
@@ -56,6 +56,44 @@ Web Debug Toolbar
 
 When using a client configured with ``HttplugBundle``, you will get debug information in the web debug toolbar. It will tell you how many request were made and how many of those that were successful or not. It will also show you detailed information about each request.
 
+You can configure the bundle to show debug information for clients found with discovery. You may also force a specific client to be found when a third party library is using discovery. The configuration below makes sure the client with service id ``httplug.clients.my_guzzle5`` is returned when calling ``HttpClientDiscovery::find()`` . It does also make sure to show debug info for asynchronous clients.
```

Comment:
i would move the topic of auto discovery to the end of the debug toolbar section, with a heading one level below web debug toolbar, to make it more clear this is about an edge case. and i would mention in the intro of this subsection that explicit DI is preferred.